Makefile

```
@@ -39,20 +39,29 @@ help: ## Show this help message
-	@echo "Building webui pages (homepage + error)..."
-	@cd webui && npm install --silent && npm run build
-	@echo "Frontend build complete"
-
-build-quick: ## Build without frontend (faster for backend development)
-	@echo "Building $(BINARY_NAME) $(VERSION) (quick, no frontend)..."
+generate-systemviews: ## Generate Go code from systemviews templ components
+	@echo "Generating systemviews templ components..."
+	go tool templ generate -path internal/ui/systemviews -include-version=false
+	@echo "Systemviews templ generation complete"
+
+build-systemviews: generate-systemviews ## Build the WASM systemviews runtime
```

checkmake (0.3.2)

[warning] Target body for "build-systemviews" exceeds allowed length of 5 lines (7).

(maxbodylength)